### PR TITLE
Add devolo home network integration

### PR DESCRIPTION
**Pull request by Shutgun**
## Proposed change

Add docs for new devolo home network integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/45866
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/2227
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by Shutgun** [comment]
The linting error comes from the product name containing a lower case "ac". However, the product is called that way. Can I overrule that somehow?

**Comment by frenck** [comment]
The only solution is to remove the word from the text lint dictionary you can find in the root of this repository.

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Comment by Shutgun** [comment]
Sry @frenck , I might have screwed up this PR. Could you please tell me, if I need to take any action?

**Comment by frenck** [comment]
Seems to be a rebase? Nothing looks out of the ordinary to me?

**Comment by Shutgun** [comment]
Yes, I was preparing a different change and it seems I used the next branch in my fork for this one here which was not to clever ...

Thanks for checking.

File: source/_integrations/devolo_home_network.markdown [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: devolo Home Network
+description: Instructions on how to integrate devolo Home Network devices with Home Assistant.
+ha_category: 
+  - Sensor
+ha_release: '2021.3'
```
Comment (frenck) [inline source/_integrations/devolo_home_network.markdown]:
```suggestion
ha_release: '2021.12'
```

File: source/_integrations/devolo_home_network.markdown [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+ha_quality_scale: platinum
+---
+
+The devolo Home Network integration allows you to monitor your PLC network.
+
+## Configuration
+
+Menu: **Configuration** -> **Integrations**.
+
+devolo Home Network devices are automatically discovered. If not, add them via the add integration menu. In this case you need to know the IP address of the device.
```
Comment (frenck) [inline source/_integrations/devolo_home_network.markdown]:
```suggestion
{% include integrations/config_flow.md %}
```

**Review by frenck — APPROVED** [review]

**Pull request merged**